applications/GeoMechanicsApplication/tests/C-Phi_reduction_process/MaterialParameters_stage1.json

```diff
@@ -0,0 +1,47 @@
+{
+   "properties": [{
+      "model_part_name":         "PorousDomain.Material_1",
+      "properties_id":           1,
+      "Material": {
+          "constitutive_law": {
+              "name"             :  "GeoLinearElasticPlaneStrain2DLaw" 
+          },
+          "Variables": {
+              "IGNORE_UNDRAINED"         :  true,
```

Comment:
Should IGNORE_UNDRAINED be set to true?

Reply:
I changed it on false and the solution is the same. It seems it shall be just to have correct data reading.